index.css

```diff
@@ -87,12 +155,13 @@ button:hover {
 }
 
 .results-container {
-    margin-top: 20px;
+    max-width: 500px;
```

Comment:
This does not seem to be enough space for a 5x sticker craft on a 1440p resolution.